eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -21,19 +21,18 @@ jobs:
       displayName: Check Smoke Test Eligibility
       steps:
         # pipelines only supports template each when it generates the entire script block.
-        - ${{ each artifact in parameters.Artifacts }}:
-          - ${{ if and(ne(variables['Skip.Release'], 'true'), ne(artifact.skipPublishPackage, 'true')) }}:
-            - pwsh: |
-                $deps = (Select-Xml -Path ./common/SmokeTests/SmokeTest/SmokeTest.csproj `
-                                    -Xpath //ItemGroup/PackageReference/@Include).Node.Value
-                if ($deps.Contains("${{ artifact.name }}")) {
-                  Write-Host "Smoke tests will run for ${{ artifact.name }}"
-                  Write-Host "##vso[task.setvariable variable=RunSmokeTests]true"
-                } else {
-                  Write-Host "Smoke tests will NOT run for ${{ artifact.name }}"
-                }
-              name: check_smoke_tests_${{ artifact.safeName }}
-              displayName: Check smoke test eligibility for ${{ artifact.name }}
+        - ${{ if and(ne(variables['Skip.Release'], 'true'), ne(parameters.Artifact.skipPublishPackage, 'true')) }}:
```

Comment:
Lets ensure that we are correctly scoping to the exact package we are releasing.

Reply:
Yeah, it should be already scoped to the exact released package.

Downloaded Version:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1046302&view=logs&j=b61764e4-2b38-5b4a-c599-d8da1081f4df&t=cd24fc2b-7cf2-5fb4-71f5-499205e750fb&l=162

Released Version:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1046302&view=logs&j=8a59aa26-f1cb-5dea-999b-76ab4b63e138&t=009a8b76-fd91-58d4-0e63-224c4153136f&l=515

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1046302&view=artifacts&pathAsName=false&type=publishedArtifacts